src/containers/Action/CountrySection/index.js

```diff
@@ -36,15 +45,17 @@ const CountrySection = () => {
                 {allActions.map((data, i) => (
                     <div ref={refs[i]} key={data.id} id={data.id}>
                         <CountryWorkCard
-                            title={data.title}
-                            organization={data.organization}
-                            description={data.summary}
-                            contextTitle={data.contextTitle}
-                            modalDescription={data.description}
+                            item={data}
+                            toggleWorkModal={() => handleToggle(data)}
```

Comment:
this creates a new object with each render. So decreases the performance. Not recommended